projects/libs/core-library-typescript/src/common-library/grpc-server.ts

```diff
@@ -1,10 +1,18 @@
 import * as Mali from "mali";
```

Comment:
general comment, maybe make grpc server output a log message with everything it knows / when starting